locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -936,8 +937,7 @@ protected MachineLocation obtainOnce(ConfigBag setup) throws NoMachinesAvailable
                     } else {
                         LOG.warn("Using DEPRECATED flag OPEN_IPTABLES (will not be supported in future versions) for {} at {}", machineLocation, this);
 
-                        @SuppressWarnings("unchecked")
-                        Iterable<Integer> inboundPorts = (Iterable<Integer>) setup.get(INBOUND_PORTS);
+                        Iterable<Integer> inboundPorts = Ints.asList(template.getOptions().getInboundPorts());
 
                         if (inboundPorts == null || Iterables.isEmpty(inboundPorts)) {
```

Comment:
Very minor: we could now get rid of the null check, given `Ints.asList()` (and `template.getOptions().getInboundPorts()`) will never return null.

Also, we don't need to convert it to an `Iterable<Integer>`. We could leave it as a `int[]`, and change the line below to `inboundPorts.length == 0`. No strong feelings.

But will merge as-is.